packages/repository/src/__tests__/unit/repositories/relations-helpers/mongo-related-helpers.unit.ts

```diff
@@ -0,0 +1,135 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/repository-tests
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {ObjectID} from 'bson';
+import {belongsTo, Entity, hasMany, model, property} from '../../../..';
+import {
+  isBsonType,
+  deduplicate,
+  normalizeKey,
+  buildLookupMap,
+  reduceAsArray,
+} from '../../../../relations';
+
+describe('unit tests, simulates mongodb env for helpers of ininclusion resolver ', () => {
+  describe('helpers for formating instances', () => {
+    it('isBsonType', () => {
+      const id = new ObjectID();
+      expect(isBsonType(id)).to.be.true();
+    });
+
+    context('deduplicate + isBsonType', () => {
```

Comment:
how are you testing `isBsonType` in these tests?

Reply:
`normalizeKey` calls `isBsontype`

Reply:
where are you calling `normalizeKey` in these tests?